quarkus/runtime/src/main/java/org/keycloak/quarkus/runtime/cli/Picocli.java

```diff
@@ -276,10 +279,20 @@ public static void validateNonCliConfig(List<String> cliArgs, AbstractCommand ab
         List<String> ignoredBuildTime = new ArrayList<>();
         List<String> ignoredRunTime = new ArrayList<>();
         Set<String> deprecatedInUse = new HashSet<>();
+        Set<String> disabledInUse = new HashSet<>();
+
         for (OptionCategory category : abstractCommand.getOptionCategories()) {
             List<PropertyMapper> mappers = new ArrayList<>();
             Optional.ofNullable(PropertyMappers.getRuntimeMappers().get(category)).ifPresent(mappers::addAll);
             Optional.ofNullable(PropertyMappers.getBuildTimeMappers().get(category)).ifPresent(mappers::addAll);
+
+            if (options.includeBuildTime) {
+                Optional.ofNullable(PropertyMappers.getDisabledBuildTimeMappers().get(category)).ifPresent(mappers::addAll);
+            }
+            if (options.includeRuntime && !isRebuildCheck()) {
```

Comment:
The reason behind this is to prevent warnings about disabled runtime mappers from being displayed in the log of the JVM managing the reaugmentation. Because when the reaug is needed for `start` or `start-dev`, the `includeRuntime=true` in this case. That might be a little bit confusing, but that's the current case in Keycloak.  

I haven't investigate it more yet, but I'll take a look at the possible options.